hs_restclient/endpoints/resource.py

```diff
@@ -27,6 +27,19 @@ def move_or_rename(self, payload):
         r = self.hs._request('POST', url, None, payload)
         return r
 
+    def zip(self, payload):
+        url = "{url_base}/resource/{pid}/functions/zip/".format(

```

Comment:
I think we need docstring for each of these api functions documenting function parameters and return type with example data to make the api easier for the users to use.